changing the cisco asa version

AhmadAlDeiriMSFT opened the pull request:
cisco asa VTI for ike v2 is supported in 9.8+ not 8.4+
the reference is https://www.cisco.com/c/en/us/td/docs/security/asa/asa98/release/notes/asarn98.html 
So we need a cisco asa OS to be above 9.8

azurecla commented:
Hi __@mariusrusu__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

davidwrede commented:
@mariusrusu, thanks for the fix!

AhmadAlDeiriMSFT committed:
changing the cisco asa version

cisco asa VTI for ike v2 is supported in 9.8+ not 8.4+
the reference is https://www.cisco.com/c/en/us/td/docs/security/asa/asa98/release/notes/asarn98.html 
So we need a cisco asa OS to be above 9.8
